### ISSUE TITLE
cardScanningEnabled to false change nothing

### ISSUE THREAD

**Issue by djacques33** [stripe/stripe-ios#1751]
Hello Team,

## Summary
I need to disable scan card button in add card view but when i set to false cardScanningEnabled in STPPaymentConfiguration the button not disapear. The only issue for remove this button is remove NSCameraUsageDescription in info.plist but i need this for other usage.

## Code to reproduce
```
STPPaymentConfiguration.shared.cardScanningEnabled = false
self.paymentContext = STPPaymentContext(customerContext: self.customerContext)
self.paymentContext.presentPaymentOptionsViewController()
```

## SDK version
21.2.0

## Other information
In STPAddCardViewController i see the lines where the button is created : 
  ```
func setUpCardScanningIfAvailable() {
    if #available(iOS 13.0, *) {
      if !STPCardScanner.cardScanningAvailable() || configuration?.cardScanningEnabled == nil {
        return
      }
      let scannerCell = STPCardScannerTableViewCell()
      self.scannerCell = scannerCell

      let cardScanner = STPCardScanner(delegate: self)
      cardScanner.cameraView = scannerCell.cameraView
      self.cardScanner = cardScanner

      cardHeaderView?.buttonHidden = false
      cardHeaderView?.button?.setTitle(
        STPLocalizedString("Scan Card", "Text for button to scan a credit card"), for: .normal)
      cardHeaderView?.button?.addTarget(self, action: #selector(scanCard), for: .touchUpInside)
      cardHeaderView?.setNeedsLayout()
    }
  }
```
cardScanningEnabled is not used for display cause is not nil this value is false but i can't set to nil in my code is Bool var.
### ###

**Comment by davidme-stripe** [comment]
Hey @djacques33, sorry about that! We didn't translate this line correctly from Objective-C. We'll issue a fix soon.

**Comment by davidme-stripe** [comment]
This was fixed in [21.2.1](https://github.com/stripe/stripe-ios/releases/tag/21.2.1).
